src/sentry/static/sentry/app/components/contextData.jsx

```diff
@@ -73,6 +72,39 @@ function analyzeStringForRepr(value) {
   return rv;
 }
 
+class ToggleWrap extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {toggled: false};
+  }
+
+  render() {
+    if (this.props.children.length === 0) {
```

Comment:
Is children always an array?

Reply:
Nope, great catch. Will use the `React.Children.count()` helper.